pkg/local_object_storage/metabase/VERSION.md

```diff
@@ -5,6 +5,7 @@ This file describes changes between the metabase versions.
 ## Current
 
 Numbers stand for a single byte value.
+The lowest not used bucket index: 20.
```

Comment:
incrementing this is one more thing to support and remember, dont think it's worth it

Reply:
Yeah, they will get out of sync quickly.

Reply:
i do not like this file in general. can you suggest something?